regolith/schemas.py

```diff
@@ -972,6 +972,542 @@
         "title": "Dr.",
     },
     ],
+    "people": [
```

Comment:
something seems wrong here.   There should already be a people collection in the schema so you shouldn't be adding it.  What are you trying to do here?

Reply:
@sbillinge we were trying to add Songsheng as a "current template person" to the schemas.py to be able to run the test file accordingly. We wanted to have Anthony Schopatz for the former and Songsheng for the current.

Reply:
pls see my comment below